miden/src/cli/run.rs

```diff
@@ -66,7 +66,26 @@ impl RunCmd {
         let trace = processor::execute(&program, stack_inputs, advice_provider, execution_options)
             .map_err(|err| format!("Failed to generate execution trace = {:?}", err))?;
 
-        println!("done ({} steps in {} ms)", trace.get_trace_len(), now.elapsed().as_millis());
+        println!("done ({} ms)", now.elapsed().as_millis());
+        println!(
+            "VM cycles: {} extended to {} steps.
+├── Stack rows: {}
+├── Range checker rows: {}
+└── Chiplets rows: {}
+    ├── Hash chiplet rows: {}
+    ├── Bitwise chiplet rows: {}
+    ├── Memory chiplet rows: {}
+    └── Kernel ROM rows: {}",
+            trace.trace_len_summary().trace_len(),
+            trace.trace_len_summary().padded_trace_len(),
+            trace.trace_len_summary().main_trace_len(),
+            trace.trace_len_summary().range_trace_len(),
+            trace.trace_len_summary().chiplets_trace_len().trace_len(),
+            trace.trace_len_summary().chiplets_trace_len().hash_chiplet_len(),
+            trace.trace_len_summary().chiplets_trace_len().bitwise_chiplet_len(),
+            trace.trace_len_summary().chiplets_trace_len().memory_chiplet_len(),
+            trace.trace_len_summary().chiplets_trace_len().kernel_rom_len(),
+        );
```

Comment:
Let's move output above VM cycles (the way I had it in the example).

Also, let's add padding ratio to the first line - for example:
```
VM cycles: 12454 extended to 223453 steps (20% padding).
```